tests/loading_from_clipboard/loading_from_clipboard_test.cpp

```diff
@@ -0,0 +1,128 @@
+#include "gtest/gtest.h"
+#include "loading_from_clipboard_test.h"
+#include <QTextStream>
+#include "../../common/decklist.h"
+
+DeckList *fromClipboard(QString *clipboard);
+DeckList *fromClipboard(QString *clipboard) {
+    DeckList *deckList = new DeckList;
+    QTextStream *stream = new QTextStream(clipboard);
+    deckList->loadFromStream_Plain(*stream);
+    return deckList;
+}
+
+using CardRows = QMap<QString, int>;
+
+struct DecklistBuilder {
+    CardRows actualMainboard;
+    CardRows actualSideboard;
+
+    explicit DecklistBuilder() : actualMainboard({}), actualSideboard({}) {}
+
+    void operator()(const InnerDecklistNode *innerDecklistNode, const DecklistCardNode *card) {
```

Comment:
Prefer using lambdas over function objects when you can. I think in this case you should be able to.